src/xrGame/memory_space.h

```diff
@@ -143,8 +143,8 @@ struct CVisibleObject : CMemoryObject<CGameObject>
 
 public:
     IC CVisibleObject() { m_visible.zero(); }
-    IC bool visible(const squad_mask_type& mask) const { return (!!m_visible.test(mask)); }
-    IC void visible(const squad_mask_type& mask, bool value) { m_visible.set(mask, value ? TRUE : FALSE); }
+    [[nodiscard]] bool visible(const squad_mask_type& mask) const { return (!!m_visible.test(mask)); }
```

Comment:
Make it more readeable, reduce code dancing horizontally...
```suggestion
    [[nodiscard]]
    bool visible(const squad_mask_type& mask) const { return (!!m_visible.test(mask)); }
```